src/fidesops/ops/schemas/connection_configuration/connection_secrets_saas.py

```diff
@@ -58,9 +58,9 @@ def get_saas_schema(self) -> Type[SaaSSchema]:
                 if connector_param.default_value
                 else (str, ...)
             )
-        SaaSSchema.__doc__ = f"{str(self.saas_config.type).capitalize()} secrets schema"  # Dynamically override the docstring
+        SaaSSchema.__doc__ = f"{str(self.saas_config.type).capitalize()} secrets schema"  # Dynamically override the docstring to create a description
         model: Type[SaaSSchema] = create_model(
-            f"{self.saas_config.fides_key}_schema",
+            f"{self.saas_config.type}_schema",
```

Comment:
This was added because we now have the saas_config templates fides_keys as <instance_fides_key> which would cause validation to fail when we're trying to build schemas.